### ISSUE TITLE
BUG: Not working well with CORS

### ISSUE THREAD

**Issue by chneau** [vermaysha/elysia-compress#3]
```ts
import cors from "@elysiajs/cors";
import { compression } from "elysia-compress";

const app = new Elysia()
	.use(serverTiming())
	.use(cors())
	.use(compression({ as: "global", threshold: 0 }))
```

But it's working if I put the threshold to 0. The http request is coming from another port using vite + react + @elysiajs/eden

Let me know if you need more information :)

**Comment by vermaysha** [comment]
Thanks, i will fix it ASAP

**Comment by vermaysha** [comment]
Fixed #4

**Comment by eokic** [comment]
I don't know if this issue resurfaced in the meantime but I'm in the same situation, and only `threshold: 0` fixes the issue.

version 1.2.1
